qatopics.md

```diff
@@ -0,0 +1,74 @@
+## Main Endpoint
+**GET /api/integration/qatopics**
+
+Provide access to the Quality Assurance Broker topics. It returns the list of the Quality Assurance Broker topics.
+
+```json
+[
+
+  {
+    id: "ENRICH!MORE!PID",
+    type: "qatopic",
+    name: "ENRICH/MORE/PID",
+    lastEvent: "2020/10/09 10:11 UTC",
+    totalSuggestions: "33"
+  },
+  {
+    id: "ENRICH!MISSING!ABSTRACT",
+    type: "qatopic",
+    name: "ENRICH/MISSING/ABSTRACT",
+    lastEvent: "2020/10/09 10:11 UTC",
+    totalSuggestions: "21"
+  },
+  ...
+]
+```
+Attributes:
+* name: the name of the topic to display on the frontend user interface
+* lastEvent: the date of the last update from Quality Assurance Broker
+* totalEvents: the total number of suggestions provided by Quality Assurance Broker for this topic
```

Comment:
Again, don't use "suggestions", use "events".  And again, we may want to note that the list of events can be found via "qaevents"